app/Http/Livewire/Importer.php

```diff
@@ -252,14 +302,14 @@ public function mount()
         $this->authorize('import');
         $this->progress = -1; // '-1' means 'don't show the progressbar'
         $this->progress_bar_class = 'progress-bar-warning';
-        \Log::debug("Hey, we are calling MOUNT (in the importer-file) !!!!!!!!"); //fcuk
         $this->importTypes = [
             'asset' =>      trans('general.assets'),
             'accessory' =>  trans('general.accessories'),
             'consumable' => trans('general.consumables'),
             'component' =>  trans('general.components'),
             'license' =>    trans('general.licenses'),
             'user' =>       trans('general.users'),
+            'location' =>    trans('general.locations'),
```

Comment:
I added it to the bottom of the list because it's less common than the others. It's far more common for users to import all of those other first-class objects versus locations, so I was trying to list it by priority, not alphabetical.

Lemme chew on that though.